projects/ngx-d3timeline/src/lib/axis/axis-utils.ts

```diff
@@ -26,6 +27,10 @@ export function getAxisLine(
   return createLine(viewTopLeft, getAxisEndPoint(viewTopLeft, orientedScale));
 }
 
+export function getGridLineWidth(orientedScale: OrientedScale<Scale>) {
```

Comment:
Looks to me like its the length, plus that is what the parameter in createOrientedLine is that it gets passed to.

```suggestion
export function getGridLineLength(orientedScale: OrientedScale<Scale>) {
```